x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/AppliedQueryRules.java

```diff
@@ -13,23 +13,16 @@
 import static org.elasticsearch.xpack.searchbusinessrules.PinnedQueryBuilder.Item;
```

Comment:
not related to your changes, but do you think the name `Item` still fits our purposes? My guess is that when we added the pinned query, we did not think this class will get reused, so having it called `Item` has not an issue.
But now when I see `Item` referenced outside of the pinned query, I have to stop and think where that comes from.

Reply:
That's a good question. `Item` is nice because it's short, but `PinnedDocument` would probably be a better name in retrospect since `Document` would be too overloaded. Or `PinnedDoc` if we thought `PinnedDocument` was way too long. 

I'd be open to a quick rename PR as a followup.